app/helpers/layout_helper.rb

```diff
@@ -27,7 +27,11 @@ def submit_title
   end
 
   def invoices_debt(debtor)
-    str = "#{debtor.name}:"
+    if debtor.is_a? Customer
+      str = "#{debtor.name}:"
```

Comment:
@dissident add html escaping. This string is defined as html safe on return, but customer name is not html safe.